docs/md/expressions.md

```diff
@@ -262,25 +260,16 @@ components: scheme, path, and display (optional). For example:
  - _twitterid:54784326227#nyaruka_
  - _telegram:34642632786#bobby_
 
-It has several properties which can be accessed in expressions:
-
- * `scheme` the scheme of the URN, e.g. "tel", "twitter"
- * `path` the path of the URN, e.g. "+16303524567"
- * `display` the display portion of the URN, e.g. "+16303524567"
- * `channel` the preferred [channel](#context:channel) of the URN
-
 To render a URN in a human friendly format, use the [format_urn](expressions.html#function:format_urn) function.
 
 Examples:
 
 
 ```objectivec
 @(contact.urns[0]) → tel:+12065551212
-@(contact.urns[0].scheme) → tel
-@(contact.urns[0].path) → +12065551212
-@(contact.urns[1].display) → nyaruka
+@(urn_parts(contact.urns[0]).scheme) → tel
```

Comment:
Since these examples are used in completion, should we update these all to use `@urns` as we think that's the more typical case?

Reply:
only docs on functions are used for auto-completion but I can update docs for `format_urns(..)` to only show `@urns` and update this to show both.